libs/model/test/contest/contests-projection-lifecycle.spec.ts

```diff
@@ -63,8 +63,17 @@ describe('Contests projection lifecycle', () => {
     getContestScore = Sinon.stub(contestHelper, 'getContestScore');
 
     await bootstrap_testing();
+    const abi = await models.ContractAbi.create({
```

Comment:
seed should take care of this - either as aggregate or entity within aggregate

Reply:
I didn't see any example of seeding ContractAbi, maybe I missed something. It didn't seem like an aggregate and I didn't want to spend a bunch of time figuring out how to make it a child of an aggregate.

Reply:
ok, let's talk about this later... related to the other question about consistency boundaries... btw, make sure you merge master

Reply:
Will do this as a fast follow